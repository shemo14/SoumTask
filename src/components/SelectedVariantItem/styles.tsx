import {StyleSheet} from 'react-native';
export default StyleSheet.create({
  container: {
    backgroundColor: '#999',
    padding: 5,
    margin: 2,
  },
  text: {
    color: '#fff',
  },
});
