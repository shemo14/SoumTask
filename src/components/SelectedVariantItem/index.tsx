import React from 'react';
import {Text} from '../../common';
import {View} from 'react-native';
import styles from './styles';

const SelectedVariantItem = ({item}: {item: string}) => {
  return (
    <View style={styles.container}>
      <Text type={'p3'} style={styles.text}>
        {item}
      </Text>
    </View>
  );
};

export default SelectedVariantItem;
