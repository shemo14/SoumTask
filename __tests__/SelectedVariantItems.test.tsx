import React from 'react';
import 'react-native';
import SelectedVariantItem from '../src/components/SelectedVariantItem';
import {it, expect} from '@jest/globals';
import renderer from 'react-test-renderer';

it('renders SelectedVariantItem snapShot correctly', () => {
  const snapShot = renderer.create(<SelectedVariantItem item={'test item'} />).toJSON();
  expect(snapShot).toMatchSnapshot();
});
