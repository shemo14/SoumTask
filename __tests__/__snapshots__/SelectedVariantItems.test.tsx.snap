// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SelectedVariantItem snapShot correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#999",
      "margin": 2,
      "padding": 5,
    }
  }
>
  <Text
    style={
      [
        {
          "fontFamily": "Roboto_Regular",
          "fontSize": 16,
          "lineHeight": 22,
          "paddingTop": 0,
        },
        {
          "color": "#fff",
        },
      ]
    }
    type="p3"
  >
    test item
  </Text>
</View>
`;
